src/libraries/System.Private.CoreLib/src/System/Runtime/InteropServices/NativeMemory.cs

```diff
@@ -38,6 +38,15 @@ public static unsafe partial class NativeMemory
             return AllocZeroed(byteCount, elementSize: 1);
         }
 
+        /// <summary>Clears a block of memory.</summary>
+        /// <param name="ptr">A pointer to the block of memory that should be cleared.</param>
+        /// <param name="byteCount">The size, in bytes, of the block to clear.</param>
+        [CLSCompliant(false)]
+        public static unsafe void ZeroMemory(byte* ptr, nuint byteCount)
```

Comment:
The first argument should be `void*` to match the approved shape.

Reply:
Whoops, typo! 😅
Fixed in 0fa37e6fbe25c399286a387e913295913e2999b7.